cms/envs/common.py

```diff
@@ -2771,3 +2774,13 @@
 #### django-simple-history##
 # disable indexing on date field its coming django-simple-history.
 SIMPLE_HISTORY_DATE_INDEX = False
+
```

Comment:
This affects the CMS API swagger docs but not the legacy swagger docs under /api-docs/.

Reply:
(Add as comment)